msteams-platform/TOC.yml

```diff
@@ -376,20 +376,20 @@
     href: /MicrosoftTeams/tenant-apps-catalog-teams?toc=/microsoftteams/platform/toc.json&bc=/MicrosoftTeams/breadcrumb/toc.json
   - name: Publish to the Teams store (AppSource)
     items:
-    - name: Understanding AppSource submission 
+    - name: Overview
       href: concepts/deploy-and-publish/appsource/publish.md
-    - name: Pre-submission checklist
-      href: concepts/deploy-and-publish/appsource/prepare/submission-checklist.md
-    - name: App detail page checklist
-      href: concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md
-    - name: App manifest checklist
-      href: concepts/deploy-and-publish/appsource/prepare/app-manifest-checklist.md
-    - name: Frequently failed cases
+    - name: Review store validation guidelines
       href: concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md
-    - name: What to do after publishing your app
+    - name: Create app documentation
+      href: concepts/deploy-and-publish/appsource/prepare/create-app-docs.md
+    - name: Create a store listing
+      href: concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md
+    - name: Prepare your store submission
+      href: concepts/deploy-and-publish/appsource/prepare/submission-checklist.md
+    - name: Submit your app
+      href: concepts/deploy-and-publish/appsource/submit-your-app.md
```

Comment:
@heath-hamilton - Remove this content & instead embed - **Store step-by-step submission guide** 
https://docs.microsoft.com/en-us/office/dev/store/add-in-submission-guide

Reply:
section on PC account creation, Category & FAQ moved to prepare for Store submission